tests/fixtures/test_vectors.json

```diff
@@ -0,0 +1,621 @@
+{
+    "general": {
+        "MNEMNONIC": "acoustic trophy damage hint search taste love bicycle foster cradle brown govern endless depend situate athlete pudding blame question genius transfer van random vast",
+        "MNEMNONIC_ADDRESS": "iota1qpg2xkj66wwgn8p2ggnp7p582gj8g6p79us5hve2tsudzpsr2ap4skprwjg"
+    },
+    "python": {
```

Comment:
do we really need language specific vectors?

Reply:
@samuel-rufi it's currently used only for the Python tests so there not two files need to be imported and at least I don't plan to adapt the tests in the other languages to the same format
For stardust I would also not continue like this

Reply:
just for better understanding, you plan to remove the test_vectors.json in future completely?

Reply:
It was never added like this there, I only added the addresses https://github.com/iotaledger/iota.rs/blob/develop/tests/fixtures/test_vectors.json

Reply:
okay i see thanks for explaining